src/libman/Commands/RestoreCommand.cs

```diff
@@ -20,8 +23,14 @@ public RestoreCommand(IHostEnvironment hostEnvironment, bool throwOnUnexpectedAr
 
         protected override async Task<int> ExecuteInternalAsync()
         {
+            var sw = new Stopwatch();
+            sw.Start();
+
             Manifest manifest = await GetManifestAsync();
-            await ManifestRestorer.RestoreManifestAsync(manifest, Logger, CancellationToken.None);
+            IEnumerable<ILibraryOperationResult> results = await ManifestRestorer.RestoreManifestAsync(manifest, Logger, CancellationToken.None);
```

Comment:
Can you call IEnumerable<ILibraryOperationResult> results = await manifest.RestoreAsync(cancelToken); directly?

Reply:
The `ManifestRestorer` does the same. The only thing it does additionally is to log failures on the console.

Reply:
I see.. I am making changes so the Logger also logs the errors.